src/transformations/transformations.ts

```diff
@@ -186,6 +186,14 @@ export interface ContentChangeTransformation {
   diff: PositionDiff;
 }
 
+export interface ExecuteNormalTransformation {
+  type: 'executeNormal';
+  keystroke: string;
```

Comment:
Shouldn't this be called `keystrokes`? (as well as `NormalCommand.keystrokes`?)

Reply:
Fixed on 
e70e7ea